dashboard.el

```diff
@@ -438,6 +443,12 @@ Optional argument ARGS adviced function arguments."
           (delete-char -1)
           (insert dashboard-page-separator))
         (dashboard-insert-footer)
+        (when dashboard-vertically-center-content
+          (goto-char (point-min))
+          (let* ((buffer-lines (count-lines (point-min) (point-max)))
+                 (vertical-padding (floor (/ (- (window-height) buffer-lines) 2))))
+            (when (> vertical-padding 0)
+              (insert (make-string vertical-padding ?\n)))))
```

Comment:
This won't work since the line height isn't an accurate calculation. Try calculating it in pixels, see below:

```elisp
        (when dashboard-vertically-center-content
          (goto-char (point-min))
          (when-let* ((content-height (cdr (window-absolute-pixel-position (point-max))))
                      (vertical-padding (floor (/ (- (window-pixel-height) content-height) 2)))
                      ((> vertical-padding 0))
                      (vertical-lines (/ vertical-padding (line-pixel-height))))
            (insert (make-string vertical-lines ?\n))))
```

Reply:
Thanks for the suggestion!